vspace/src/main/java/edu/asu/diging/vspace/references/ReferenceDisplayProvider.java

```diff
@@ -0,0 +1,57 @@
+package edu.asu.diging.vspace.references;
+
+import org.springframework.stereotype.Service;
+
+import edu.asu.diging.vspace.core.model.impl.Reference;
+
+@Service
+public class ReferenceDisplayProvider {
```

Comment:
There needs to be an interface for this class

Reply:
the presentation of references needs to be as flexible as the metadata embedding